doc/Learn/Maui/ThirdParty-DevExpress.md

```diff
@@ -50,20 +86,75 @@ Once you have an account with DevExpress, you need to visit the [Your DevExpress
                 .UseDevExpress()
                 .ConfigureFonts(fonts =>
                 {
-                    fonts.AddFont("MauiEmbeddingApp/Assets/Fonts/OpenSansRegular.ttf", "OpenSansRegular");
-                    fonts.AddFont("MauiEmbeddingApp/Assets/Fonts/OpenSansSemibold.ttf", "OpenSansSemibold");
+                    fonts.AddFont("Assets/Fonts/OpenSansRegular.ttf", "OpenSansRegular");
+                    fonts.AddFont("Assets/Fonts/OpenSansSemibold.ttf", "OpenSansSemibold");
                 });
     }
     ```
```

Comment:
I think the snippet there isn't showing as it should:

![image](https://github.com/unoplatform/uno.extensions/assets/114886335/dd71d8f2-4318-4ab0-9950-5c61096fa70e)